services/net/content/ContentManager.cs

```diff
@@ -167,6 +167,9 @@ private bool ConsumerErrorHandler(object sender, ErrorEventArgs e)
         this.State.RecordFailure();
         if (e.GetException() is ConsumeException ex)
         {
+            // Need to tell Kafka that this means it can't continue.
+            if (ex.Message == "Broker: Unknown topic or partition")
```

Comment:
Force a failure that will stop the consuming thread.  I'm not 100% this is the behaviour we want, but I also don't want the service pretending it's running.